src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -388,6 +390,22 @@ export class ResultsViewPageStore {
                     }, false);
 
                 return anySelected;
+            },
+
+            get driverFilter() {
+                return AppConfig.serverConfig.oncoprint_custom_driver_annotation_binary_default;
+            },
+
+            set driverFilter(val:boolean) {
+                this._driverFilter = val;
+            },
```

Comment:
I'm not completely sure what you're going for but I think what you want is to change the initial value of _driverFilter to be undefined on line 359. Then `get driverFilter()` should return ` this._driverFilter === undefined ? AppConfig... : this._driverFilter`.